routes/stats.js

```diff
@@ -52,10 +52,18 @@ statsRouter.get('/week', async (req, res) => {
     const response = await pool.query(
       `SELECT SUM(pounds) AS pound_sum, SUM(ounces) AS ounces_sum FROM event_data INNER JOIN events on event_data.event_id=events.id WHERE events.date BETWEEN CURRENT_DATE - INTERVAL '7 days' AND CURRENT_DATE`,
     );
-    const pounds = parseFloat(response.rows[0].pound_sum);
-    const ounces = parseFloat(response.rows[0].ounces_sum);
-    const total = (pounds + ounces / 16).toFixed(2);
-    res.json(total);
+    const pounds = response.rows[0].pound_sum != null ? parseFloat(response.rows[0].pound_sum) : 0;
+    const ounces = response.rows[0].ounces_sum != null ? parseFloat(response.rows[0].ounces_sum) : 0;
+    const total = (pounds + ounces / 16).toFixed();
+
+    const response2 = await pool.query(
+      `SELECT SUM(pounds) AS pound_sum, SUM(ounces) AS ounces_sum FROM event_data INNER JOIN events on event_data.event_id=events.id WHERE events.date BETWEEN CURRENT_DATE - INTERVAL '14 days' AND CURRENT_DATE - INTERVAL '7 days'`,
+    );
+    const poundsTwo = response.rows[0].pound_sum != null ? parseFloat(response2.rows[0].pound_sum): 0;
+    const ouncesTwo = response.rows[0].ounces_sum != null ? parseFloat(response2.rows[0].ounces_sum) : 0;
+    const totalTwo = poundsTwo + ouncesTwo / 16;
+    const rate = (totalTwo != 0 ? (((total - totalTwo) / totalTwo) * 100) : total).toFixed(1);
```

Comment:
🚫 **[eslint]** <[eqeqeq](https://eslint.org/docs/latest/rules/eqeqeq)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Expected '!==' and instead saw '!='.